src/main/webapp/user/UserRepresentation.js

```diff
@@ -0,0 +1,25 @@
+class UserRepresentation {
+  constructor(user) {
+    this._userRepresentationDiv = createUserDiv(user);
+  }
+
+  get userRepresentationDiv() {
+    return this._userRepresentationDiv;
+  }
+}
+
+function createUserDiv(user) {
+  const userDiv = document.createElement("div");
+  const userDivClass = "userRepresentation-" + "{" + user.id() + "}";
+  userDiv.setAttribute("class", userDivClass);
+  userDiv.addEventListener("click", function() {
+    if (user.publicPortfolio) {
+      var url = new URL("/portfolio.html", document.URL);
+      url.searchParams.append('user', user)
+      window.location = url;
+    } else {
+      alert("The creator of this place guide does not have a private place guide!");
+    }
+  });
+  return userDiv;
+}
```

Comment:
This won't work. First, /portfolio.html will show the user's own portfolio. Plus, UserRepresentation doesn't display the user icon or anything related to the user. Let's leave it for a separate PR.
You can keep a sketch in order not to break other code.

Reply:
removed user representation.